x11-misc/sddm/files/sddm-0.12.0-consolekit.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/data/scripts/Xsession b/data/scripts/Xsession
```

Comment:
This file is the same as files/sddm-0.11.0-consolekit.patch and files/sddm-0.10.0-consolekit.patch, please, leave only one of them.